### PR TITLE
app-backup/deja-dup: bump to 40.6, remove old, and remove mask

### PR DESCRIPTION
**Pull request by petronio**
Closes: https://bugs.gentoo.org/show_bug.cgi?id=710928
Package-Manager: Portage-2.3.94, Repoman-2.3.21
Signed-off-by: Petronio Coelho <petronio@petroniocoelho.com>

### CODE REVIEW

File: app-backup/deja-dup/Manifest [hunk 1]
```diff
@@ -1 +1 @@
-DIST deja-dup-34.3.tar.xz 702708 BLAKE2B 219bb3796f3b97b240f9ac353dfb3c206c6423e8b8c8d4e5a97c89d832226ad70cfb1116bbd900bac1fc1bf03608aad47c67af0441fda7bc8fe950313fa14ebf SHA512 0e703c18041fa5ca6d2f272f5b0e65ad31ff1094147432107834d1cd25e2ec14b87bdfac6faf727c97339d20d1681f3251c5f74b6edbee4368014839b1f04f06
```
Comment (mgorny) [inline app-backup/deja-dup/Manifest:1]: First of all, split removal into a separate commit.

File: app-backup/deja-dup/deja-dup-40.6.ebuild [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```
Comment (mgorny) [inline app-backup/deja-dup/deja-dup-40.6.ebuild:4]: I don't think there's anything preventing you from using EAPI 7.

File: app-backup/deja-dup/deja-dup-40.6.ebuild [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+inherit meson eutils gnome2-utils vala xdg
```
Comment (mgorny) [inline app-backup/deja-dup/deja-dup-40.6.ebuild:6]: Sort when possible. Also since you're overriding all the phases, `xdg-utils`, not `xdg`. Where are `eutils` used?

File: app-backup/deja-dup/deja-dup-40.6.ebuild [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+DESCRIPTION="Simple backup tool using duplicity back-end"
+HOMEPAGE="https://gitlab.gnome.org/World/deja-dup"
+SRC_URI="https://gitlab.gnome.org/World/${PN}/-/archive/${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-3"
```
Comment (mgorny) [inline app-backup/deja-dup/deja-dup-40.6.ebuild:12]: Are you sure it's not GPL-3+?

File: app-backup/deja-dup/deja-dup-40.6.ebuild [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="nautilus test"
+RESTRICT="!test? ( test )"
+
+COMMON_DEPEND="
```
Comment (mgorny) [inline app-backup/deja-dup/deja-dup-40.6.ebuild:18]: Given that the whole thing is written in vala, are you sure you don't need introspection flags everywhere?

File: app-backup/deja-dup/metadata.xml [hunk 6]
```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+  <maintainer type="person">
+    <email>petronio@petroniocoelho.com</email>
+    <name>Petronio Coelho</name>
+  </maintainer>
   <maintainer type="project">
```
Comment (mgorny) [inline app-backup/deja-dup/metadata.xml:8]: Have you discussed this with GNOME project? If not, you should remove them and add proxy-maint instead.

**Comment by ajakk** [comment]
Well, unfortunately this package was removed in the meantime.
